.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Node.js ${{ matrix.node-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version:
+          - 20
+          - 18
+          - 16
```

Comment:
- 20: current
- 18: lts
- 12: the version that was supposedly compatible when this package was being developed